pack/tcu/bbt_encounter.json

```diff
@@ -0,0 +1,404 @@
+[
+	{
+		"back_text": "Hard / Expert\n[skull]: -X. X is the amount of doom on Azathoth, to a minimum of 2.\n[cultist]: Reveal another token. If you fail, search the encounter deck and discard pile for a [[Cultist]] enemy and draw it. Shuffle the encounter deck.\n[tablet]: -3. If you fail, Azathoth attacks you.\n[elder_thing]: -6. If your modified skill value for this test is 0, place 1 doom on Azathoth.",
+		"code": "05025",
+		"double_sided": true,
+		"encounter_code": "before_the_black_throne",
+		"faction_code": "mythos",
```

Comment:
Please add "encounter_position" to all cards.

Reply:
Done